src/Raven.Server/Documents/Sharding/Handlers/ShardedIncomingReplicationHandler.cs

```diff
@@ -36,8 +39,20 @@ public class ShardedIncomingReplicationHandler : AbstractIncomingReplicationHand
             : base(tcpConnectionOptions, buffer, parent.Server, parent.DatabaseName, replicatedLastEtag, parent.Context.DatabaseShutdown, parent.Server.ContextPool)
         {
             _parent = parent;
+
+            DevelopmentHelper.ShardingToDo(DevelopmentHelper.TeamMember.Shiran, DevelopmentHelper.Severity.Normal, "figure out if the sharded db is encrypted ('encrypted' parameter for 'TempFileCache')");
```

Comment:
I don't understand what is required here... you mean `parent.Server.Configuration.Security.IsCertificateConfigured` is wrong and we need to figure out encryption in some other way? check the record for `IsEncrypted`?

Reply:
IDK if it's wrong. What do you think? :-)
It is true for encrypted db, but I'm not sure if it is good enough